docs/source/index.rst

```diff
@@ -7,6 +7,13 @@ Dallinger allows crowd sourced experiments to be abstracted into single function
 
 The Dallinger technology stack consists of: Python, Redis, Web Sockets, Heroku, AWS, Mechanical Turk, Prolific, boto, Flask, PostgreSQL, SQLAlchemy, Gunicorn, Pytest and gevent among others.
 
+.. admonition:: Subscribe to our Slack channel
+    :class: support-slack-channel-box
+
+    To help users with technical issues and experiment creation, the Dallinger team maintains a **Dallinger support Slack channel**. Access to the Dallinger support Slack requires a **$10 per month per user subscription**, with proceeds supporting the future development of Dallinger. Users are free to subscribe for as many or as few months as they wish.
```

Comment:
Can we modify the beginning of the paragraph like this? 

> Dallinger is open-source, and is completely free to use in your experiments. However, to help users with technical issues and experiment creation [...]

Reply:
Done.